addons/l10n_vn/data/template/account.tax-vn.csv

```diff
@@ -11,6 +11,10 @@
 "","","","","","","","","tax","invoice","","",""
 "","","","","","","","","base","refund","-Untaxed Purchase of Goods and Services taxed 0%","",""
 "","","","","","","","","tax","refund","","",""
+"tax_purchase_vat_exemption","VAT Exemption","VAT Exemption","VAT Exemption","0.0","percent","purchase","tax_group_exemption","base","invoice","+Untaxed Purchase of Goods and Services taxed VAT Exemption","","Không thuộc đối tượng chịu thuế GTGT"

```

Comment:
For exemption taxes, Odoo's taxonomy is to name them like this: `0% VAT EXEMPT`. (both for the `name` and `invoice_label` field)

could you rename those?

Reply:
Yes why not done